emulated_hue/api.py

```diff
@@ -557,19 +557,6 @@ async def async_get_timezones(self, request: web.Request):
         """Return all timezones."""
         return send_json_response(self.config.definitions["timezones"])
 
-    # Static Content Begin
-    @routes.get("/clip.html")
-    @check_request(False)
-    async def async_get_clip_debugger(self, request: web.Request):
-        """Serve the CLIP Debugger."""
-        return web.Response(text=self._clip_html, content_type="text/html")
-
-    @routes.get("/robots.txt")
-    @check_request(False)
-    async def async_get_robots_txt(self, request: web.Request):
-        """Serve robots.txt."""
-        return web.Response(text=const.ROBOTS_TXT, content_type="text/plain")
```

Comment:
Seems like robots.txt went missing?

Reply:
see my earlier comment. we should create a static file ronots.txt for that

Reply:
Is robots.txt present in the official bridge? We would also need to remove ROBOTS_TXT from the const file as well.